ci/artifacts/xline-pod.yaml

```diff
@@ -6,12 +6,52 @@ metadata:
   namespace: kube-system
 spec:
   containers:
-    - name: xline
-      image: ghcr.io/xline-kv/xline:master
-      imagePullPolicy: IfNotPresent
-      command:
-        - /bin/sh
-        - -c
-        - xline --name node1 --members node1=kind-control-plane:2379 --data-dir /tmp/xline --storage-engine rocksdb --client-listen-urls=http://kind-control-plane:2379 --peer-listen-urls=http://kind-control-plane:2380,http://kind-control-plane:2381 --client-advertise-urls=http://kind-control-plane:2379 --peer-advertise-urls=http://kind-control-plane:2380,http://kind-control-plane:2381
+  - name: xline
+    image: ghcr.io/xline-kv/xline:master
+    imagePullPolicy: IfNotPresent
+    livenessProbe:
+      failureThreshold: 8
+      httpGet:
+        host: 127.0.0.1
+        path: /metrics
+        port: 9100
+        scheme: HTTP
+      initialDelaySeconds: 10
+      periodSeconds: 10
+      timeoutSeconds: 15
+    startupProbe:
+      failureThreshold: 24
+      httpGet:
+        host: 127.0.0.1
+        path: /metrics
```

Comment:
This probably requires a better one, but I haven't found one. Remind me if i missed something,Thanks.

Reply:
Basically, this requires the application to have `/readyz` and `/livez` interfaces. etcd is also implement the grpc for it https://github.com/etcd-io/etcd/commit/293f087fb987421348dd9a4173087bb5f08cb850#diff-ab6fb0684315e16355f6ebe0f4b3cf860b9b2ff5a0fe1b4e4308a680b19f1b0c